external/src/main/java/edu/illinois/cs/cogcomp/pipeline/handlers/PathLSTMHandler.java

```diff
@@ -117,20 +117,20 @@ private PredicateArgumentView getSRL(TextAnnotation ta) throws Exception {
                 List<String> relations = new ArrayList<>();
 
                 for (Word a : p.getArgMap().keySet()) {
-
                     Set<Word> singleton = new TreeSet<Word>();
                     String label = p.getArgumentTag(a);
                     Yield y = a.getYield(p, label, singleton);
                     IntPair span = new IntPair(y.first().getIdx() - 1, y.last().getIdx());
 
                     assert span.getFirst() <= span.getSecond() : ta;
-                    args.add(new Constituent(label, viewName, ta, span.getFirst(), span.getSecond()));
+                    if(pav.getConstituentsCoveringSpan(span.getFirst(), span.getSecond()).size() == 0)
```

Comment:
This seems like incorrect behavior: shouldn't you confirm that one constituent in the returned list has the same start and end offsets as the one you want to add? it seems like this will incorrectly skip many constituents.